aioauth/utils.py

```diff
@@ -268,8 +270,11 @@ async def wrapper(self, request, *args, **kwargs) -> Response:
                     status_code=HTTPStatus.FOUND,
                     headers=HTTPHeaderDict({"location": location}),
                 )
-            except Exception:
-                error = ServerError(request=request)
+            except Exception as exc:
+                error = ServerError(
+                    request=request,
+                    description=str(exc) if request.settings.DEBUG else "",
```

Comment:
In projects I've worked on previously, I found it was easier and more convenient to simply check the logging level of the logger rather than have some sort of separate global flag. that way you can just configure the logger to allow debug rather than anyone having to be aware of the project-specific setting.

something like this:
```python3
description=str(exc) if log.level <= logging.DEBUG else ""
```

that's just a suggestion tho.